hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMClientProtocolServer.java

```diff
@@ -783,6 +783,27 @@ public ScmInfo getScmInfo() throws IOException {
     }
   }
 
+  @Override
+  public int resetDeletedBlockRetryCount(List<Long> txIDs) throws IOException {
+    Map<String, String> auditMap = Maps.newHashMap();
+    getScm().checkAdminAccess(getRemoteUser());
+    try {
+      int count = scm.getScmBlockManager().getDeletedBlockLog().
+          resetCount(txIDs);
+      scm.getScmHAManager().asSCMHADBTransactionBuffer().flush();
```

Comment:
Why flush is required here?

Reply:
This manually triggers the batch operation flushing into RDB, otherwise,  it wouldn't take effect after the next natural flush (e.g `take snapshot`/ ` 'download SCM' ), which is uncontrollable.

Reply:
I think SCM will handle the flush itself.  We don't need to control it.  Unless flush is a must to have here, I will recommend to remove it. Otherwise, if user frequently call this API, SCM performance can be impacted.